interface-definitions/interfaces_wireless.xml.in

```diff
@@ -428,6 +419,139 @@
+                      <valueHelp>
+                        <format>133</format>
+                        <description>80 MHz channel width</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>134</format>
+                        <description>160 MHz channel width</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>135</format>
+                        <description>80+80 MHz channel width</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>(131|132|133|134|135)</regex>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <node name="center-channel-freq">
+                    <properties>
+                      <help>HE operating channel center frequency</help>
+                    </properties>
+                    <children>
+                      <leafNode name="freq-1">
+                        <properties>
+                          <help>HE operating channel center frequency - center freq 1 (for use with 80, 80+80 and 160 modes)</help>
+                          <valueHelp>
+                            <format>u32:1-233</format>
+                            <description>6Ghz (802.11 ax) center channel index (use 3 (at 40MHz), 7 (at 80MHz) or 15 (at 160MHz) for primary channel 1)</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-233"/>
+                          </constraint>
+                          <constraintErrorMessage>Channel center value must be between 1 and 233</constraintErrorMessage>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="freq-2">
+                        <properties>
+                          <help>HE operating channel center frequency - center freq 2 (for use with the 80+80 mode)</help>
+                          <valueHelp>
+                            <format>u32:1-233</format>
+                            <description>6Ghz (802.11 ax) center channel index (use 23 (at 80MHz) for secondary channel 17)</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-233"/>
+                          </constraint>
+                          <constraintErrorMessage>Channel center value must be between 1 and 233</constraintErrorMessage>
+                        </properties>
+                      </leafNode>
+                    </children>
+                  </node>
+                  <leafNode name="antenna-pattern-fixed">
+                    <properties>
+                      <help>Set if antenna pattern does not change during the lifetime of an association</help>
```

Comment:
I feel really uneasy about this wording, to be fair. Ideally, the help string should tell the user what effect the option has. Let's think about possible options.

Reply:
I have addressed your remarks in commit [74736d4](https://github.com/vyos/vyos-1x/pull/3524/commits/74736d4ca0e7e00799abe03d3f78a43eeee457e9).